lavamoat/browserify/main/policy.json

```diff
@@ -695,6 +695,7 @@
     },
```

Comment:
I'm not a 100% sure how to review these, but it looks like changes match up with changes made to `skunkworks` 👍